trade.md

```diff
@@ -9,14 +9,62 @@ gcam-version: v5.3
 
-## Armington Style Trade for Primary Ag
-For the primary agriculture commodities such as Corn, Rice, and purpose grown bio-energy we use an Armington style distinction between domestic and imported goods, but not with fixed bilateral imports.    The competition between imports and domestic is governed by a logit.  Imports are from a single global pool that draws from all regions and is also governed by a logit.
+The Heckscher-Ohlin theorem explains trade using factor endowments and predicts that each country produces goods with more intensive use of its abundant factor of production (Peter Debaere, 2003; Vanek, 1968). The empirical use of the Heckscher-Ohlin approach assumes products are homogeneous across sources and traded in a single global market (i.e., fully integrated world market). Markets clear at the world level and each region will see the same global price and independently decide how much each will supply and demand of each commodity given that price. A region's net trade position is dynamic depending on economics, technical change, demand, growth, resources, etc.  Under this method for trading goods there is no modeled preference for a given region to demand a commodity from any other specific region. 
+
+The trade of agricultural and livestock proudcts were mostly modeled using the Heckscher-Ohlin approach in early version of GCAM (e.g., GCAM v4). But it has been updated to the Armington style trade modeling approach for most of the agricultural and livestock proudcts. Fodder_herb is modeled using the Heckscher-Ohlin approach and Fodder_grass is not traded. Also, major energy commodities such as coal, gas, oil, bio-energy, etc. are also traded in a single world market with the Heckscher-Ohlin approach.      
+
+
+## Armington Style Trade
+For the agricultural and livestock commodities in GCAM (except fodder crops and fish & other meats), we use an Armington style distinction between domestic and imported goods. The Armington approach assumes products are differentiated by source and consumers view goods produced in different countries as imperfect substitutes (Armington, 1969). The theoretical background and the derivation of the logit-based Armington approach is documented in Zhao et al. (2020). In this approach, the competition between imports and domestic is governed by a logit. Imports are from a single global pool that draws from all regions and is also governed by a logit. The logit-based Armington approach requires a segmented regional market, as oppose to the integrated world market in the Heckscher-Ohlin approach. Thus, it allows differentiating regional prices and tracing gross trade flows. 
+
+
+The structural implementations of a "global-market" versus a "regional-market" representation are shown in Figure 1 with an example of corn trade.
+
+![AgLU Global Markets](gcam-figs/ag_global_mkt.png)<br/>
+![AgLU Regional Markets](gcam-figs/ag_regional_mkt.png)<br/>
```

Comment:
These figures didn't render when I built the website; Figure 2 does, so perhaps switch to that way of referring to the image file?